integrations/localhero/src/main/java/ch/stayhome/integrations/localhero/config/LocalHeroProperties.java

```diff
@@ -9,31 +10,45 @@
 import lombok.Data;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 @ConfigurationProperties(prefix = "stayhome.integrations.localhero")
-@EnableConfigurationProperties({LocalHeroProperties.class})
 @Data
 public class LocalHeroProperties {
-    @NotBlank(message = "No cron expression has been defined in application.yml")
-    private String scrapeCron;
 
-    @NotEmpty
-    private List<String> sourceUrls;
-    
-    @NotBlank(message = "No rest route has been defined in application.yml")
-    private String restRoute;
```

Comment:
@lugipfupf  I was not able to use this in combination with the index.php?restRoute=XXXX and the pagination -> Thus I hard-coded the value directly into the LocalHeroChApi.java. It's also where I defined the route to resolve the categories ```/wp-json/wp/v2/categories/```